### PR TITLE
CDS-56851-selective-stage=execution-using-triggers

### PR DESCRIPTION
**Pull request by roshnisarangadharan**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/platform/11_Triggers/selective-stage-execution-using-triggers.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+* You have set the service and infrastructure as runtime inputs so that you can provide the value in **Trigger>** **Configuration** tab.
+
+## Execute a specific pipeline stage using triggers
+
+1. Select a pipeline whose stage you want to execute using triggers.
+2. Make sure that the **Selective Execution Settings>** **[Allow selective stage(s) executions?](/docs/platform/pipelines/run-specific-stage-in-pipeline/)** in **Advanced Options** of the pipeline is set to **Yes**.
+   
+    ![](./static/selective-stage-execution.png)
+
+3. When you create a [custom trigger](/docs/platform/triggers/trigger-deployments-using-custom-triggers/), in the **Configuration** tab, select the stages you want to execute.
```
Comment (vcalasansh) [inline docs/platform/11_Triggers/selective-stage-execution-using-triggers.md:28]: The selective stage execution is available for all types of triggers, not only custom triggers. So I think you replace "custom trigger" with "trigger" here.

Comment (roshnisarangadharan) [inline docs/platform/11_Triggers/selective-stage-execution-using-triggers.md:28]:
```suggestion
3. When you create a trigger, in the **Configuration** tab, select the stages you want to execute.
```

File: docs/platform/11_Triggers/selective-stage-execution-using-triggers.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+---
+title: Selective pipeline stage execution by using triggers
+description: Execute specific stages of a pipeline using custom triggers
```
Comment (roshnisarangadharan) [inline docs/platform/11_Triggers/selective-stage-execution-using-triggers.md:3]:
```suggestion
description: Execute specific stages of a pipeline using webhook triggers
```

File: docs/platform/11_Triggers/selective-stage-execution-using-triggers.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+:::note
+
+Currently, this feature is behind the feature flag, `CDS_NG_TRIGGER_SELECTIVE_STAGE_EXECUTION`. Contact [Harness Support](mailto:support@harness.io) to enable the feature.
+
+:::
+
+## Requirements
+
+Make sure you have:
```
Comment (roshnisarangadharan) [inline docs/platform/11_Triggers/selective-stage-execution-using-triggers.md:17]:
```suggestion
Make sure that you have met the following requirements:
```

File: docs/platform/11_Triggers/selective-stage-execution-using-triggers.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+:::note
+
+Currently, this feature is behind the feature flag, `CDS_NG_TRIGGER_SELECTIVE_STAGE_EXECUTION`. Contact [Harness Support](mailto:support@harness.io) to enable the feature.
+
+:::
+
+## Requirements
+
+Make sure you have: 
+* A Harness pipeline with a [CD stage](https://developer.harness.io/docs/continuous-delivery/onboard-cd/cd-concepts/cd-pipeline-basics).
```
Comment (roshnisarangadharan) [inline docs/platform/11_Triggers/selective-stage-execution-using-triggers.md:18]:
```suggestion
* You have a Harness pipeline with a [CD stage](https://developer.harness.io/docs/continuous-delivery/onboard-cd/cd-concepts/cd-pipeline-basics).
```

**Review by charanya-j — APPROVED** [review]

**Pull request merged**